docs/scoring.md

```diff
@@ -23,58 +23,57 @@ All audits in the SEO category are [equally weighted](https://github.com/GoogleC
 
 ## How is the accessibility score calculated?
 
-<!--
-  sum = auditRefs.reduce((sum, item) => sum += item.weight, 0);
-  auditRefs
-    .filter(a => a.weight)
-    .sort((a, b) => b.weight - a.weight)
-    .map(a => [undefined, a.id, `${(a.weight / sum * 100).toLocaleString(undefined, {maximumFractionDigits:1})}%`, undefined].join(' | '))
-    .join('\n')
-  -->
+<!-- node lighthouse-core/scripts/print-a11y-scoring.js -->
 
-The accessibility score is a weighted average. The specific weights, at the time of publishing, are [as follows](https://github.com/GoogleChrome/lighthouse/blob/080c6b4b9fec6dfcaf8e0cd8d09c3224465e4fd3/lighthouse-core/config/default-config.js#L450-L491):
+<!-- TODO: need a commit first to update url ... -->
```

Comment:
reminder note to update this before landing, we can do this without merging :)

Reply:
I meant to delete this comment. I edited the paragraph. wdyt?

Reply:
yeah wfm, remove TODO then 👍 :)